packages/logger-plugin/tests/helpers/setup.ts

```diff
@@ -0,0 +1,30 @@
+import { ErrorHandler } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { NgxsModule, Store } from '@ngxs/store';
+import { StateClass } from '@ngxs/store/internals';
+
+import { LoggerSpy } from './logger-spy';
+import { NoopErrorHandler } from '../../../store/tests/helpers/utils';
+import { NgxsLoggerPluginModule, NgxsLoggerPluginOptions } from '../../';
+
+export function setup(states: StateClass[], opts?: NgxsLoggerPluginOptions) {
```

Comment:
The method name `setup` is very common to use within tests, so could we change this name to be more specific to the type of setup so that it does not conflict with the convention (and potentially cause conflicts with the `setup` method in tests where we would want to build on top of this)?
Obviously the usages will also have to change with this rename.
```suggestion
export function setupWithLogger(states: StateClass[], opts?: NgxsLoggerPluginOptions) {
```